03.bowling/bowling.rb

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ruby
+
+require 'debug'
+
+# 引数をとる
+score = ARGV[0]
+scores = score.split(',')
+
+# 数字に変換
+shots = []
+scores.each do |s|
+    if s == 'X' # strike
+        shots << 10
+        shots << 0
+    else
+        shots << s.to_i
+    end
+end
+
+# 2フレームごとに分割する
+frames = []
+shots.each_slice(2) do |s|
+    frames << s
+end
+
+# 10フレーム目は2投もしくは3投でカウント
+if ! frames[10].nil?
+    if frames[9] == [10, 0] #10フレーム目1投目がストライクの場合、次の投球を0本にせず、もう2投分カウントする
+        if frames[10] == [10, 0] # 10フレーム目2投目もストライクの場合、次の投球を0本にせず、もう1投分カウントする
+            frames[10][1] = frames[11][0]
+            frames.pop
+        end
+        frames[9].pop
+        frames[9] << frames[10][0]
+        frames[9] << frames[10][1]
+
+    else
+        frames[9] << frames[10][0]
+    end
+    frames.pop
+end
+
+# 得点を合計する
+point = 0
+frames_array_num = 0
+
+# テスト表示用
+# pre_point = 0
+
+frames.each do |frame|
+    point += frame.sum 
+    if frames_array_num <= 8 # 10フレーム目はスペア・ストライクボーナス無しのため除外
+        if frame[0] == 10 # strike
+            point += frames[frames_array_num + 1][0]
+            if frames[frames_array_num + 1][0] == 10 && frames_array_num != 8 #9フレーム目のストライクは10フレーム目1•2投目を参照するため除外
+                point += frames[frames_array_num + 2][0]
+            else
+                point += frames[frames_array_num+1][1]
```

Comment:
```suggestion
                point += frames[frames_array_num + 1][1]
```

のように演算子の前後には空白を入れることが多いです。

Reply:
ケアレスミスでした！修正しました！
気づいてくださってありがとうございます！